Week1/app.js

```diff
@@ -0,0 +1,205 @@
+    let repositoryDescriptionData = createHTMLElement('td', repositoryDescriptionRow);
+    repositoryDescriptionTitle.innerText = 'Description : ';
+    if (repositoryInfo.description) {
+      repositoryDescriptionData.innerText = repositoryInfo.description;
+    } else {
+      repositoryDescriptionData.innerText = 'No description found';
+    }
+
+    //Addressing Fork
+    let repositoryForkRow = createHTMLElement('tr', repositoryTableBody);
+    let repositoryForkTitle = createHTMLElement('td', repositoryForkRow);
+    let repositoryForkData = createHTMLElement('td', repositoryForkRow);
+    repositoryForkTitle.innerText = 'Forks : ';
+    if (repositoryInfo.forks) {
+      repositoryForkData.innerText = repositoryInfo.forks;
+    } else {
+      repositoryForkData.innerText = '0';
+    }
+
+    //Addressing Updated
+    let repositoryUpdatedRow = createHTMLElement('tr', repositoryTableBody);
+    let repositoryUpdatedTitle = createHTMLElement('td', repositoryUpdatedRow);
+    let repositoryUpdatedData = createHTMLElement('td', repositoryUpdatedRow);
+    repositoryUpdatedTitle.innerText = 'Updated : ';
+    repositoryUpdatedData.innerText = repositoryInfo.updated_at.replace(/T/g, ' ').replace(/Z/g, '');
+
+
+
+    function repositoryInformation() {
+      newList.addEventListener('change', function () {
+        const repositoryInfo = someData[newList.value];
+        repositoryNameLink.innerText = repositoryInfo.name;
+        repositoryNameLink.setAttribute('href', repositoryInfo.html_url);
+        repositoryNameLink.setAttribute('target', '_blank');
+        repositoryNameLink.innerText = repositoryInfo.name;
+        repositoryForkData.innerText = repositoryInfo.forks;
+        repositoryUpdatedData.innerText = repositoryInfo.updated_at.replace(/T/g, ' ').replace(/Z/g, '');
+        if (repositoryInfo.description) {
+          repositoryDescriptionData.innerText = repositoryInfo.description;
+        } else {
+          repositoryDescriptionData.innerText = 'No description found';
+        }
+        if (repositoryInfo.forks) {
+          repositoryForkData.innerText = repositoryInfo.forks;
+        } else {
+          repositoryForkData.innerText = '0';
+        }
+      })
+
+    }
+    repositoryInformation();
+  }
+
+
+  //Making some HTML elements for contributor part
+  let contributorsInformationBox = createHTMLElement('div', mainDiv);
+  contributorsInformationBox.setAttribute('id', 'con_div');
+  let contributorsParagraph = createHTMLElement('p', contributorsInformationBox);
+  contributorsParagraph.innerText = 'Contributions';
+  let contributorsList = createHTMLElement('ul', contributorsInformationBox);
+
+
+
+
+  function renderContributorsInformationToDom() {
+
+    let hyfRepositoryAtTheMoment = repositoryNameLink.innerText;
+    const gitHubUrl = 'https://api.github.com/repos/HackYourFuture/' + hyfRepositoryAtTheMoment + '/contributors';
```

Comment:
You should not put the URL inside functions and in multiple places, it can get confusing really quickly